src/shared/components/DateSeparatedInputs/useDateTimeInputs.tsx

```diff
@@ -0,0 +1,157 @@
+import * as React from 'react';
```

Comment:
I think this component and the one above (DateSeparated.tsx) can be easily unit tested. They have a bunch of small logic and different cases that cannot be easily tested manually. I'm not going to block the PR for this though, but maybe it's something we should start considering.